ddsc/ddsclient.py

```diff
@@ -193,13 +193,14 @@ def run(self, args):
         """
         project_name_or_id = self.create_project_name_or_id_from_args(args)
         folder = args.folder                # path to a folder to download data into
+        resume = args.resume                # are we resuming a download
         # Default to project name with spaces replaced with '_' if not specified
         if not folder:
-            fixed_path = replace_invalid_path_chars(project_name_or_id.value.replace(' ', '_'))
-            folder = path_does_not_exist_or_is_empty(fixed_path)
+            folder = replace_invalid_path_chars(project_name_or_id.value.replace(' ', '_'))
```

Comment:
Didn't catch this before, but why replace spaces with underscores?

Reply:
> I would expect resuming a download to be enabled by default.
I wanted to avoid surprising users who expect the existing functionality.  Downloading with resume will replace existing files (can be destructive). Uploading will just make new versions so nothing is ever lost. That being said consistency with upload is appealing.

> why replace spaces with underscores?
There is some discussion here https://github.com/Duke-GCB/DukeDSClient/pull/26